src/components/Login/Login.js

```diff
@@ -13,6 +13,29 @@ class Login extends Component {
       password:''
     }
   }
+  localLogin = (e) => {
```

Comment:
Could you explain what you are trying to do here? I do not understand your intentions.